### PR TITLE
Jhony/new microsite model implementation

### PR DESCRIPTION
**Pull request by jfavellar90**
This PR enables the usage of the new microsites model created in ednx_microsites app. manage_api and microsite_api are now consuming this new model. I bumped the version in this PR to start using versioning and solve the problem we have with the installation of this app (it is not being updated)
@felipemontoya 
@diegomillan

### CODE REVIEW

File: edunext_openedx_extensions/ednx_microsites/backends/database.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
     from microsite_configuration.backends.base import BaseMicrositeBackend
 except ImportError:
     from .base import BaseMicrositeBackend
-from microsite_configuration.models import Microsite
+from ..models import Microsite
```
Comment (felipemontoya) [inline edunext_openedx_extensions/ednx_microsites/backends/database.py]: should we use the long name here? to make it more explicit?

Comment (jfavellar90) [inline edunext_openedx_extensions/ednx_microsites/backends/database.py]: I was using relative imports only for module dependencies inside the same app (in this case ednx_microsites). There are another relative imports in edunext, microsite_api and manage_api app. Do you agree with doing this change across all EOE in this PR? or better create other PR, and then rebase this one?

Comment (felipemontoya) [inline edunext_openedx_extensions/ednx_microsites/backends/database.py]: I think we should stick to absolute imports. It is the default in python 3 and the preferred way to do imports in the community and pep-328. For the scope of this PR, I say we should only change it here, and later we will make a different PR changing all the remaining relative imports.

**Comment by jfavellar90** [comment]
Ok, changes done @felipemontoya

**Pull request merged**